### PR TITLE
ISPN-5940 Added CDI Remote module to Wildfly modules

### PR DESCRIPTION
**Pull request by VaclavDedik**
https://issues.jboss.org/browse/ISPN-5940

### CODE REVIEW

File: as-modules/embedded/build.xml [hunk 1]
```diff
@@ -63,7 +63,7 @@
             <maven-resource group="org.infinispan" artifact="infinispan-cachestore-jpa" />
         </module-def>
 
-        <module-def name="org.infinispan.cdi" slot="${infinispan.slot}">
+        <module-def name="org.infinispan.cdi.embedded" slot="${infinispan.slot}">
```
Comment (slaskawi) [inline as-modules/embedded/build.xml]: We would need to mention the rename here: https://github.com/infinispan/infinispan/blob/master/documentation/src/main/asciidoc/upgrading/upgrading.adoc

Comment (slaskawi) [inline as-modules/embedded/build.xml]: @tristantarrant Can we do this kind of change in this release loop?

Comment (tristantarrant) [inline as-modules/embedded/build.xml]: I think at the very least we should have an org.infinispan.cdi module that acts as an alias for embedded

Comment (VaclavDedik) [inline as-modules/embedded/build.xml]: @tristantarrant I have added a commit that creates an alias named org.infinispan.cdi targetting org.infinispan.cdi.embedded.

**Comment by tristantarrant** [comment]
Pushed, thanks

**Pull request closed without merging**